ups/validate_base.table

```diff
@@ -0,0 +1,8 @@
+setupRequired(pyyaml)
+setupRequired(numpy)
+setupRequired(astropy)
+
+setupOptional(base)
```

Comment:
I'd drop this completely. You don't even really need it optionally. No extra functionality is enabled if it is there.